src/main/java/com/minecraftabnormals/abnormals_core/core/mixin/client/CustomizeSkinScreenMixin.java

```diff
@@ -0,0 +1,31 @@
+package com.minecraftabnormals.abnormals_core.core.mixin.client;
+
+import com.minecraftabnormals.abnormals_core.client.screen.SlabfishHatScreen;
+import net.minecraft.client.GameSettings;
+import net.minecraft.client.gui.screen.CustomizeSkinScreen;
+import net.minecraft.client.gui.screen.Screen;
+import net.minecraft.client.gui.screen.SettingsScreen;
+import net.minecraft.client.gui.widget.button.Button;
+import net.minecraft.util.text.Color;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.StringTextComponent;
+import net.minecraft.util.text.TranslationTextComponent;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.ModifyVariable;
+
+@Mixin(CustomizeSkinScreen.class)
+public abstract class CustomizeSkinScreenMixin extends SettingsScreen {
```

Comment:
Does this need to be `abstract`?
`final` would work better, in my opinion.

Reply:
I'm just used to making all my mixins abstract, can do 👍